src/Menu/Item.js

```diff
@@ -13,55 +13,53 @@ governing permissions and limitations under the License.
 import React from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
-
 import '@spectrum-css/menu'
+import { CheckMarkMedium } from '../Icons'
 
 const Item = ({
   children,
   item,
   onAction,
-  isSelected = false,
-  isDisabled = false,
   isDivider = false,
-  isSectionHeading = false,
+  isHighlighted,
+  isSelected,
+  isDisabled,
+  href,
   ...props
-}) =>
-  isDivider ? (
-    <li className='spectrum-Menu-divider' role='separator' {...props} />
+}) => {
+  const Element = href ? 'a' : 'li'
+
+  return isDivider ? (
+    <li className='spectrum-Menu-divider' role='separator' />
   ) : (
-    <li
-      className={classNames([
-        { 'spectrum-Menu-item': !isSectionHeading },
+    <Element
+      className={classNames(
+        'spectrum-Menu-item',
+        { 'is-open': isHighlighted },
         { 'is-selected': isSelected },
         { 'is-disabled': isDisabled }
-      ])}
-      role={!isSectionHeading ? 'menuitem' : 'group'}
-      aria-selected={isSelected}
-      aria-disabled={isDisabled}
```

Comment:
eslint reported aria-selected is not valid on elements with role="menuitem" so that's why I removed it ...